src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystem.java

```diff
@@ -98,13 +102,14 @@ private static void createVoltageControlledBusEquations(LfBus controlledBus, Equ
             } else {
                 boolean deriveA1 = creationParameters.isPhaseControl() && branch.isPhaseController()
                     && branch.getDiscretePhaseControl().getMode() == DiscretePhaseControl.Mode.CONTROLLER;
+                boolean deriveR1 = creationParameters.isTransformerVoltageControl() && branch.isVoltageController();
```

Comment:
Somewhere in this file we should throw an exception if we have an open branch with voltage control on because
  - does not make sens?
  - not supported by equations.

Reply:
For regulating shunt later maybe?